frontend-bugs/src/main/webapp/view/Component.controller.js

```diff
@@ -190,9 +190,6 @@ sap.ui.controller(
 												case "CHECK_VERSION":
 													lib.checkvers = affectedVersions[x].affected;
 													break;
-												case "PRE_COMMIT_POM":
```

Comment:
Sure this can be removed? Please check with Serena!

Reply:
Yes, @serenaponta told me to remove this as it is not used now